bootstrap.sh

```
@@ -1,10 +1,43 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -euo pipefail
+
+# 0) Create/ensure kind cluster from cluster.yml
+if ! kind get clusters | grep -qx "kind"; then
+  kind create cluster --config=cluster.yml
+fi
+
+# 1) Wait for nodes to be Ready (best effort)
+kubectl wait --for=condition=Ready nodes --all --timeout=180s || true
```

This wait command is good, but `|| true` suppresses failures and masks a non-ready cluster. The checklist requires waiting for nodes; it's better to let the script fail on timeout or implement retries with a clear error message so bootstrap fails loudly if nodes never become Ready.